src/timeline/timeline.tsx

```diff
@@ -68,21 +71,27 @@ export const Timeline = ({
             );
         });
     };
+
     const renderItems = () =>
         items.map((item: TimelineItemProps, index) => {
-            const { title, content, statuses } = item;
+            const { title, content, statuses, variant: _variant } = item;
             const circleIndicatorTestId = baseIndicatorTestId
                 ? `circleindicator${index + 1}_div_${baseIndicatorTestId}`
                 : "circleindicator";
+            const variant =
+                (!_variant && index === 0 ? "current" : _variant) ||
+                "upcoming-active";
```

Comment:
we could reorganise the condition checks here so the logic to set the default variant is kept together

```suggestion
            const variant = _variant || (index === 0 ? "current" : "upcoming-active");
```